### DISCUSSION TITLE
Making Progress on the FRT Portal (Tips)

### DISCUSSION THREAD

**Discussion by proy01** [github/india#322] [Students]
Since a lot of people are trying to figure out how to use the FRT Portal, I thought since I've had some luck in progressing - I'll put it on here.
1. If you cannot click on on any tab other than orientation, even if you have submitted all your details, do it again. Then Log out and log back in and go to the orientation section again. You should be able to see all the links that you submitted be saved on the text bar on which you submitted the link. Also, Make sure you have submitted links for every platform they have asked for: Microsoft Learn (link to profile), Future Skills prime (email used to create account), GitHub (link to profile), LinkedIn (link to profile), AICTE (email used to create account).

2. For people who haven't received the student developer pack, relax. This thing opened up basically today and they have asked for at least 3 days for verifying your Linkedin. Please give them at least 4. This will allow them to process all the requests.

3. The support email is not an instantaneous send-reply system. Give them at least 3 working days to reply to your email, and if you still haven't received a reply, send a follow-up.

4. From what I understand from orientation and reading the discussions, the self-learning tab will update once a week. So even if it shows "In progress" or "incomplete" continue doing the course in the order shown by the self-learning tab, and once the portal updates after a week, your completed courses should ideally be updated. If you see a refresh date and a course you completed has not been updated, that's when you should email support.

5. To complete the industry session, watch the **entire thing** without fast-forwarding. You will have to let the video play up till the end of the last second and you should receive a pop-up that says _upload successful_. That will changes the status of the video from incomplete to completed.

6. The projects tab will not be opened until the 10th of October according to the FRT portal. Till then do your courses and watch the Industry session and keep making progress.

7.  The project will end up having one of the themes of the industry sessions and will be of your OWN making. You would have to use a minimum of 3 Azure services in your final project, but the rest of the submission details will be given later, so wait for that.

I hope this was helpful to at least some of y'all. 
- Pronoy Roy (A student like you)

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
